raiden/ui/cli.py

```diff
@@ -745,6 +754,20 @@ def run(ctx, **kwargs):
     print('Welcome to Raiden, version {}!'.format(get_system_spec()['raiden']))
     from raiden.ui.console import Console
     from raiden.api.python import RaidenAPI
+    from raiden.utils import address_checksum_and_decode
```

Comment:
These new imports should go to the top of the file.

Reply:
i updated